templates/solutions/ai-infrastructure/_form-ai-infrastructure.html

```diff
@@ -0,0 +1,167 @@
+<div class="p-modal" style="display: none;" id="ai-infrastructure-modal">
+  <div class="p-modal__dialog is-paper" role="dialog">
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+        <div class="row--50-50 p-strip is-shallow">
+          <div class="col">
+            <h2 class="p-heading--1">Contact Canonical</h2>
+            <p>Fill in this form and a member of our team will be in touch shortly.</p>
+          </div>
+          <div class="col">
+            <button class="p-modal__close p-modal-close-button "
+                    aria-controls="ai-infrastructure-modal"
+                    aria-label="Close active modal"
+                    style="margin-top: 0">Close</button>
```

Comment:
We actually vanilla classes for [margin](https://vanillaframework.io/docs/utilities/margin-collapse) and padding collapse so no need for the inline style here
```suggestion
            <button class="p-modal__close p-modal-close-button u-no-margin--top"
                    aria-controls="ai-infrastructure-modal"
                    aria-label="Close active modal">Close</button>
```